webcomponents/tool/src/main/frontend/packages/sakai-push-utils/test/sakai-push-utils.test.js

```diff
@@ -0,0 +1,8 @@
+import { pushSetupComplete } from '../src/sakai-push-utils.js';
+
+describe("sakai-push-utils tests", () => {
+
+  it ("sets up uccessfully", async () => {
+    pushSetupComplete.then(() => console.log("push setup complete"));
```

Comment:
```suggestion
describe("sakai-push-utils", () => {

  it ("sets up successfully", async () => {
    pushSetupComplete.then(() => console.log("push setup complete"));
```

Reply:
Though I am not sure if this test is asserting anything?

Reply:
No, it's not. It's just a sanity check at the minute. The test will fail if the imports fail etc. I need to add some more meaningful testing although it will have to involve service worker installation etc. So ... complex. I thought it was better to at least leave the shell of the test there.